assets/src/components/UserSettingSnackbar.js

```diff
@@ -25,14 +30,21 @@ function UserSettingSnackbar (props) {
     }
   }, [saved])
 
+  const snackbarDuration = Math.max(snackbarMessage.length * 200, 4000)
+  
   return (
     <Snackbar
       anchorOrigin={{
         vertical: 'bottom',
         horizontal: 'left'
       }}
+      ContentProps={{
+        'role': 'alertdialog',
+      }}
       open={savedSnackbarOpen}
-      autoHideDuration={3000}
+      autoHideDuration={snackbarDuration}
+      TransitionComponent={SlideTransition}
+      transitionDuration={{exit: 400, enter: 400}}
```

Comment:
Add spaces so it's `{{ exit: 400, enter: 400 }}`.

Reply:
fixed